store/manager.go

```diff
@@ -94,13 +94,15 @@ func (m *Manager) Get(ctx context.Context, key []byte, cm commitments.Commitment
 			return nil, errors.New("expected EigenDA backend for DA commitment type, but none configured")
 		}
 
-		var err error
-		var data []byte
+		verifyMethod, err := m.getVerifyMethod(cm.Version)
+		if err != nil {
+			return nil, fmt.Errorf("get verify method: %w", err)
+		}
 
 		// 1 - read blob from cache if enabled
 		if m.secondary.CachingEnabled() {
 			m.log.Debug("Retrieving data from cached backends")
-			data, err := m.secondary.MultiSourceRead(ctx, key, false, m.eigenda.Verify)
+			data, err := m.secondary.MultiSourceRead(ctx, key, false, verifyMethod)
```

Comment:
does this change not also need to be applied for the fallback? See https://github.com/Layr-Labs/eigenda-proxy/blob/4602d9ab31d02605761eee416cc9cc8d46fda21a/store/manager.go#L123

Reply:
I had made the fix there too, but I must have managed to undo it... good catch, thanks. [aa0f951](https://github.com/Layr-Labs/eigenda-proxy/pull/351/commits/aa0f951dff22e674cbba738ec306acdc97c82165)